### DISCUSSION TITLE
Is there a way to add a column to the source sheet?

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#970] [Q&A] [answered]
If I start from a FreqTableSheet, is there a way to add column back to the source sheet without having to figure out how to do the join?

Like if I added a rank column to the FreqTableSheet using addcol-incr, is there a command that will push that new column and values back to the source sheet?  It would be possible to do a join, but I'd like to avoid having to setup the sheets for a join. 

Likewise, is it possible to do it with a sheet created with dup-selected, even if that left other non-duped rows with null values.   It looks like a sheet created with dup-selected doesn't know the source sheet, so that would be difficult.

**Comment by saulpw** [comment, marked as answer]
Hi @frosencrantz, there isn't a way at present, but it doesn't seem like it would be too difficult to create a command that 'hoists' a column from a FreqTable to its source sheet.  You're right that a duplicated sheet doesn't know it's source at present, but we could add that easily enough--maybe the source should even be the sheet instead of the path, to avoid accidentally overwriting the source file with a subset of the data.

**Reply by frosencrantz** [reply]
Yes, that would make the FreqTable and duplicated sheets more useful in shaping data.  It was a long time before I realized that it is possible to make edits from the FreqTable and those would update the source table.

Should I add it to the wishlist?
